scopes/component/component/ui/use-component-query.ts

```diff
@@ -19,6 +20,10 @@ const componentFields = gql`
     id {
       ...componentIdFields
     }
+    aspects {
```

Comment:
add field argument to filter which aspect you want (on the backend side)

Reply:
@odedre